BitwardenShared/Core/Platform/Models/Enum/FeatureFlag.swift

```diff
@@ -65,6 +65,9 @@ enum FeatureFlag: String, CaseIterable, Codable {
     /// A feature flag for the refactor on the SSO details endpoint.
     case refactorSsoDetailsEndpoint = "pm-12337-refactor-sso-details-endpoint"
 
+    /// A feature flag for the use of new cipher permission properties.
+    case restrictItemDeletion = "pm-15493-restrict-item-deletion-to-can-manage-permission"
+
```

Comment:
⛏️ 🤔 Does this affect any item of the app, like a send or an attachment? Or does this only affect deletion of ciphers? If the latter I'd update the case name to something more explicit like `restrictCipherItemDeletion`. I would guess you wanted to match the case name with the flag string value but if we can be more explicit the better so we don't actually need to read the flag comment here to see what's being used for.
However, if you have reasons to keep using the current one please let me know.

Reply:
Yes this is a property only for ciphers, I'll update the naming to better reflect the scope of action.